canopy_dispatch.ml

```diff
@@ -66,67 +66,72 @@ module Make (S: Cohttp_lwt.Server) (C: V1_LWT.CONSOLE) (Disk: V1_LWT.KV_RO)
     | uri::[] when uri = config.Canopy_config.push_hook_path ->
       store.update () >>= fun l ->
       respond_update l
+    | "tags"::[] -> (
+      let tags = Canopy_content.tags !cache in
+      let content = Canopy_article.to_tyxml_tags tags in
+      respond_html ~headers ~title:config.Canopy_config.blog_name ~content ~updated
```

Comment:
I think at some point we should improve the cache mechanism (which is a Map other Canopy_content.t right now), abstracting thing in such a way we can cache easily other things than articles.
I'm not too sure on how to do that properly, for now this solution works fine enough, we'll see later for something more clever